CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2021-01-04 - v5.6.1
+
+#### Improvements
+
+* Accessibility improvements.
```

Comment:
Could you note the actual accessibility changes (looks like Range labels and the `setContentDescription` addition) and note the change of `ApptentiveNestedScrollView` to `NestedScrollViewHelper` here too.

Reply:
The transition from `ApptentiveNestedScrollView` to `NestedScrollViewHelper ` is more like a "private" API change and might not bring many benefits to the "public" changelog.

Reply:
I vote we merge it as-is and be more pro-active with the changelog updates in the future.

Reply:
Is there some reason you can't start making those additions now?